### PR TITLE
Logging improvements

### PR DESCRIPTION
**Pull request by diaasami**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Review by teg — DISMISSED** [review]

File: cmd/osbuild-worker/main.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ func RequestAndRunJob(client *worker.Client, acceptedJobTypes []string, jobImpls
 	logrus.Info("Waiting for a new job...")
 	job, err := client.RequestJob(acceptedJobTypes, common.CurrentArch())
 	if err == worker.ErrClientRequestJobTimeout {
-		logrus.Debugf("Requesting job timed out: %v", err)
+		logrus.Infof("Requesting job timed out: %v", err)
```
Comment (Gundersanne) [inline cmd/osbuild-worker/main.go:93]: This is actually expected behaviour, but it's ok to have it here.

**Review by Gundersanne — DISMISSED** [review]
lgtm!

File: cmd/osbuild-worker/main.go (hunk 1 above)
Comment (ondrejbudai) [reply cmd/osbuild-worker/main.go:93]: Hmm, isn't this going to spam journal? If it is, I suggest reverting this change.

**Review by ondrejbudai — CHANGES_REQUESTED** [review]
See my comment inline. I think that we don't print debug logs by default, right?

File: cmd/osbuild-worker/main.go (hunk 1 above)
Comment (Gundersanne) [reply cmd/osbuild-worker/main.go:93]: every 20s yea, and you're right we don't have great filtering mechanisms yet.

Comment (ondrejbudai) [reply cmd/osbuild-worker/main.go:93]: @diaasami WHile you are on it, can you also change https://github.com/osbuild/osbuild-composer/blob/ab3712fd928d07c9f40651e253262494ae19d843/cmd/osbuild-worker/main.go#L90 to debug? This message is like 99,9% of our logs and it actually isn't that interesting.

**Comment by diaasami** [comment]
> See my comment inline. I think that we don't print debug logs by default, right?

No we are not.

**Review by Gundersanne — APPROVED** [review]

**Review by Gundersanne — APPROVED** [review]

**Review by ondrejbudai — APPROVED** [review]
Thanks a lot!

**Pull request merged**